build-image/openhabian.pi-raspios32.conf

```diff
@@ -60,8 +60,8 @@ debugmode=off
 # Java architecture mode
 # Valid arguments: "Adopt11", "Zulu8-32", "Zulu8-64", "Zulu11-32", or "Zulu11-64"
 # Only change cached_java_opt at image build time or things might go wrong.
```

Comment:
I think a user need not know about cache_java_opt or ?
To him these dual parameters will be confusing.
(yes I know this is about OH3 not caching but nonetheless)

Reply:
That is there for build time, I need it for proper installation. I just put that comment there to avoid a user changing it

Reply:
hmm,ok